.github/workflows/api-fix-docs-artefacts.yml

```
@@ -28,9 +28,9 @@ jobs:
     steps:
       - name: Generate GitHub App token with push access
         id: app-token
-        uses: actions/create-github-app-token@v1
+        uses: actions/create-github-app-token@v3
```

actionlint (1.7.12)

[error] missing input "app-id" which is required by action "actions/create-github-app-token@v3". all required inputs are "app-id", "private-key"

(action)